magda-web-client/src/Components/RequestDataset/RequestFormTemplate.js

```diff
@@ -156,9 +156,17 @@ export default class RequestFormTemplate extends React.Component {
                     onChange={this.handleInputChange}
                     type="text"
                     placeholder={this.props.textAreaPlaceHolder}
+                    {...(this.state.messageValid
+                        ? {}
+                        : {
+                              "aria-describedby": "messageFieldDescription"
+                          })}
                 />
                 {!this.state.messageValid && (
-                    <label className="correspondence-field-error">
+                    <label
+                        id="messageFieldDescription"
```

Comment:
Can we call this id="messageFieldError"? I'd say the description is the label ;).

Reply:
All field naming have been changed from `xxxxDescription` to `xxxxError`~
Please have a look and let me know if anything else requires changes~